.circleci/config.yml

```diff
@@ -41,13 +41,22 @@ jobs:
       - run:
           name: Verify lint
           command: yarn lint
+      - run:
+          name: Build types
+          command: yarn workspace @requestnetwork/types run build
+      - run:
+          name: Build utils
+          command: yarn workspace @requestnetwork/utils run build
+      - run:
+          name: Build currency
+          command: yarn workspace @requestnetwork/currency run build
```

Comment:
Reasoning : these 3 packages are core packages potentially used in other packages, we should build them beforehand.

If I recall correctly: `smart-contracts` tests now require `Currency`, which requires `utils`, which requires `types`.